visual-testing-app/webpack.config.js

```diff
@@ -22,18 +22,41 @@ if (!info) {
 }
 
 module.exports = {
+  target: 'web',
+  // note: we have to use 'development' for now, because when we use production
+  // our emotion styles aren't displayed in our percy snapshots.
```

Comment:
Can we also mention some follow up or something, or link to an issue related to this (in case there is one)?

Reply:
I emailed support directly so there isn't a github issue. I did create a reproduction repo to show to support [here](https://github.com/montezume/percy-puppeteer-emotion).